### PR TITLE
Fix all broken summary xml docs

### PR DESCRIPTION
**Pull request by AlmightyLks**
Fixes https://github.com/elsa-workflows/elsa-core/pull/5860#issuecomment-2321225095 in all places of elsa

- - -
This change is [Reviewable](https://reviewable.io/reviews/elsa-workflows/elsa-core/6123)

### CODE REVIEW

**Comment by AlmightyLks** [comment]
@dotnet-policy-service agree

File: src/modules/Elsa.Workflows.Runtime/Filters/BookmarkQueueFilter.cs [hunk 1]
```diff
@@ -2,37 +2,57 @@
...
     /// Gets or sets the bookmark hash of the bookmark queue item to find.
+    /// </summary>
     public string? BookmarkHash { get; set; }
 
+    /// <summary>
     /// The ID of the activity instance associated with the bookmark.
+    /// </summary>
     public string? ActivityInstanceId { get; set; }
 
     // The type name of the activity associated with the bookmark.
```
Comment (Suchiman) [inline src/modules/Elsa.Workflows.Runtime/Filters/BookmarkQueueFilter.cs:40]:
```suggestion
    /// <summary>
    /// The type name of the activity associated with the bookmark.
    /// </summary>
```

File: src/modules/Elsa.Workflows.Runtime/Entities/BookmarkQueueItem.cs [hunk 2]
```diff
@@ -4,31 +4,47 @@
...
     /// A bookmark payload hash of the bookmark to resume.
+    /// </summary>
     public string? StimulusHash { get; set; }
 
+    /// <summary>
     /// The ID of the activity instance associated with the bookmark.
+    /// </summary>
     public string? ActivityInstanceId { get; set; }
 
     // The type name of the activity associated with the bookmark.
```
Comment (Suchiman) [inline src/modules/Elsa.Workflows.Runtime/Entities/BookmarkQueueItem.cs:37]:
```suggestion
    /// <summary>
    /// The type name of the activity associated with the bookmark.
    /// </summary>
```

File: src/modules/Elsa.MassTransit/Options/MassTransitOptions.cs [hunk 3]
```diff
@@ -5,13 +5,21 @@ namespace Elsa.MassTransit.Options;
...
     public TimeSpan? TemporaryQueueTtl { get; set; }
+    /// <summary>
     /// The number of concurrent messages to process.
+    /// </summary>
     public int? ConcurrentMessageLimit { get; set; }
+    /// <summary>
     /// The number of messages to fetch from the bus with each request.
+    /// </summary>
     public int? PrefetchCount { get; set; }
+    /// <summary>
```
Comment (Suchiman) [inline src/modules/Elsa.MassTransit/Options/MassTransitOptions.cs:20]:
```suggestion

    /// <summary>
```

File: src/modules/Elsa.MassTransit/Options/MassTransitOptions.cs [hunk 4]
```diff
@@ -5,13 +5,21 @@ namespace Elsa.MassTransit.Options;
...
 {
+    /// <summary>
     /// The TTL of queues that are seen as temporary (typically queues that are created per running instance).
+    /// </summary>
     public TimeSpan? TemporaryQueueTtl { get; set; }
+    /// <summary>
     /// The number of concurrent messages to process.
+    /// </summary>
     public int? ConcurrentMessageLimit { get; set; }
+    /// <summary>
```
Comment (Suchiman) [inline src/modules/Elsa.MassTransit/Options/MassTransitOptions.cs:16]:
```suggestion

    /// <summary>
```

File: src/modules/Elsa.MassTransit/Options/MassTransitOptions.cs [hunk 5]
```diff
@@ -5,13 +5,21 @@ namespace Elsa.MassTransit.Options;
 /// </summary>
 public class MassTransitOptions
 {
+    /// <summary>
     /// The TTL of queues that are seen as temporary (typically queues that are created per running instance).
+    /// </summary>
     public TimeSpan? TemporaryQueueTtl { get; set; }
+    /// <summary>
```
Comment (Suchiman) [inline src/modules/Elsa.MassTransit/Options/MassTransitOptions.cs:12]:
```suggestion

    /// <summary>
```

**Comment by AlmightyLks** [comment]
The more I look at it, the more I notice there is to fix. Not yet ready

**Comment by AlmightyLks** [comment]
Fixed the rest now. Ready for review.

**Review by sfmskywalker — APPROVED** [review]
Thank you @AlmightyLks for taking the time and doing the effort!

**Comment by sfmskywalker** [comment]
Also, a big thanks for the extra pair of eyes from @Suchiman !

**Pull request merged**